src/dataset-builder/ConceptSetCreator.test.ts

```diff
@@ -1,6 +1,7 @@
 import { screen } from '@testing-library/react';
 import userEvent from '@testing-library/user-event';
 import { h } from 'react-hyperscript-helpers';
+import { SnapshotBuilderConcept } from 'src/libs/ajax/DataRepo';
```

Comment:
From what I've seen, SnapshotBuilderConcept is imported as `Concept`. Thoughts on 
`import { SnapshotBuilderConcept as Concept } from 'src/libs/ajax/DataRepo';`